### ISSUE TITLE
Test Real-time 3D Reconstruction

### ISSUE THREAD

**Issue by chenchen2015** [chenchen2015/CameraTrackingSystem#15]
**Description**
Use the trained neural networks model from #13 to test reconstruct new gantry positions and evaluate static and dynamic reconstruction accuracy.

**TODO**
- [x] Write a MATLAB script that constantly captures camera views and processes the images to find the target (*outputs Nx2 matrix that contains 2D image coordinates of N camera views*)
- [x] Test the MATLAB script. Find a gantry position range that we can have a reliable output.
- [x] Feed the output of the image processing function to the trained neural network function to get the reconstructed gantry coordinates.
- [x] Read Gantry ground truth and compare results
- [x] Use the result to estimate static reconstruction accuracy - reconstruct N times at a given location and use the sample distribution to estimate the mean and confidence interval
- [x] Estimate dynamic reconstruction accuracy by moving the gantry unit distance away
